### ISSUE TITLE
"color for object keys" from JQ_COLORS  doesn't seem to be respected

### ISSUE THREAD

**Issue by ksowa** [jqlang/jq#3115]
**Describe the bug**
When I tried to change `color for object keys` by setting the 8th item in the `JQ_COLORS` variable, it didn't make any difference.

Manual page: https://jqlang.github.io/jq/manual/v1.7/#colors

**To Reproduce**
Create a `test.json` file with the following contents:
```
{
  "field_1": true,
  "field_2": false,
  "field_3": 123456,
  "field_4": "abc",
  "field_5": ["a", 1],
  "field_6": null
}
```

Run a few commands, using different setting for the `color for object keys` value. For example:

```
cat test.json | JQ_COLORS='1;90:0;35:0;35:0;36:0;32:1;35:1;35;1:30' jq

cat test.json | JQ_COLORS='1;90:0;35:0;35:0;36:0;32:1;35:1;35;0:35' jq

cat test.json | JQ_COLORS='1;90:0;35:0;35:0;36:0;32:1;35:1;35;1:90' jq
```

The outcome is always the same - the object keys are blue.
![image](https://github.com/jqlang/jq/assets/1027333/9566843a-5b3b-476a-ab4e-08a82a67fc07)

**Expected behavior**
I would expect the color of object keys to change when I change the 8th color in the `JQ_COLORS` variable.

All the other colors change as expected.

**Environment (please complete the following information):**

- OS and Version: `Ubuntu 22.04`
- jq version: `jq-1.6`

**Comment by itchyny** [comment]
What's the output of `jq --version`

**Comment by ksowa** [comment]
@itchyny I edited my report, as initially I put a wrong version number.

```
jq-1.6
```

**Comment by itchyny** [comment]
The `color for object keys` in `JQ_COLORS` was implemented in 1.7. Please update jq version.

**Comment by ksowa** [comment]
I just updated to `jq-1.7.1-51-gc127616` and it works as expected.

Thank you for your help!
